websites/I/India Today News/presence.ts

```diff
@@ -0,0 +1,541 @@
+						url: href,
+					},
+				],
+			},
+			"/crime": {
+				details: "Viewing crime news",
+				buttons: [
+					{
+						label: "View crime news",
+						url: href,
+					},
+				],
+			},
+			"/podcasts": {
+				details: "Viewing podcasts news",
+				buttons: [
+					{
+						label: "View podcasts news",
+						url: href,
+					},
+				],
+			},
+			"/opinion-columns": {
+				details: "Viewing opinion columns news",
+				buttons: [
+					{
+						label: "View opinion columns news",
+						url: href,
+					},
+				],
+			},
+			"/horoscopes": {
+				details: "Viewing horoscopes news",
+				buttons: [
+					{
+						label: "View horoscopes news",
+						url: href,
+					},
+				],
+			},
+			"/interactive": {
+				details: "Viewing interactive news",
+				buttons: [
+					{
+						label: "View interactive news",
+						url: href,
+					},
+				],
+			},
+			"/visualstories": {
+				details: "Viewing visual stories",
+				buttons: [
+					{
+						label: "View visual stories",
+						url: href,
+					},
+				],
+			},
+			"/cryptocurrency": {
+				details: "Viewing cryptocurrency news",
+				buttons: [
+					{
+						label: "View cryptocurrency news",
+						url: href,
+					},
+				],
+			},
+			"/fact-check": {
+				details: "Viewing fact check news",
+				buttons: [
+					{
+						label: "View fact check news",
+						url: href,
+					},
+				],
+			},
+			"/data-intelligence-unit": {
+				details: "Viewing DIU news",
+				buttons: [
+					{
+						label: "View DIU news",
+						url: href,
+					},
+				],
+			},
+			"/photo": {
+				details: "Viewing photo news",
+				buttons: [
+					{
+						label: "View photo news",
+						url: href,
+					},
+				],
+			},
+			"/news.html": {
+				details: "Viewing news",
+				buttons: [
+					{
+						label: "View news",
+						url: href,
+					},
+				],
+			},
+			"/foryou": {
+				details: "Viewing for you page",
+				buttons: [
+					{
+						label: "View for you page",
+						url: href,
+					},
+				],
+			},
+			"/cover-story": {
+				details: "Viewing cover story",
+				buttons: [
+					{
+						label: "View cover story",
+						url: href,
+					},
+				],
+			},
+			"/programmes": {
+				details: "Viewing programmes",
+				buttons: [
+					{
+						label: "View programmes",
+						url: href,
+					},
+				],
+			},
+		},
+		storyDate = document.querySelector("span.strydate")?.textContent,
+		authorTitle = document
+			.querySelector("div.Story_story__author__cJoes")
+			?.querySelector("a.jsx-99cc083358cc2e2d")?.textContent,
+		multipleAuthors = document.querySelector("div.Story_authors__OqkJZ");
+	/* deepscan-disable-next-line CONSTANT_CONDITION */
+	if (oldLang !== newLang || !strings) {
+		oldLang = newLang; /* deepscan-disable-line UNUSED_VAR_ASSIGN */
+		strings = await getStrings();
+	}
+
+	for (const [path, data] of Object.entries(pages))
+		if (pathname.includes(path)) presenceData = { ...presenceData, ...data };
+
+	if (pathname === "/") presenceData.details = "Viewing the homepage";
```

Comment:
There's a string for that
https://github.com/PreMiD/Localization/blob/4d554a4766ea07673ce52548b521eb77b3d9a59d/src/Presence/general.json#L166-L169